electric-operator.el

```diff
@@ -560,9 +560,10 @@ Using `cc-mode''s syntactic analysis."
 
 (defun c-mode-\" ()
   "Handle the opening quote of an include directive"
-  (if (c-mode-include-line-opening-quote?)
-      " \""
-    "\""))
+  (cond
+   ((c-mode-include-line-opening-quote?) " \"")
+   ((in-docs?) (match-string 0))
```

Comment:
I really don't like the idea of using `match-string` here: it relies on implementation details of other functions and it relies on nothing else using regexes between wherever the match string is coming from and this line. Also it makes the code much more difficult to understand: you need to follow the control flow from much higher up to figure out which match is being used.

Does this do something different than returning `nil` here would do?

Reply:
Agree, wasn't aware that returning nil has the same effect, should have taken a close look at the function.